packages/web3/test/integration/web3.test.ts

```diff
@@ -75,6 +75,16 @@ describe('Web3 instance', () => {
 		expect(() => new Web3()).not.toThrow();
 	});
 
+	it('check disconnect function', async () => {
```

Comment:
can you test other `supportedproviders` on top of `clienturl` and check functions exist 👍

Reply:
done